requirements/base.txt

```diff
@@ -62,7 +62,9 @@ cryptography==41.0.7
     #   mozilla-django-oidc
     #   pyopenssl
 css-inline==0.13.0
-    # via -r requirements/base.in
+    # via
+    #   -r requirements/base.in
+    #   mail-editor
```

Comment:
This is probably from another PR? Doesn't actually change anything though, so it's fine

Reply:
Yes I rebased on [https://github.com/maykinmedia/open-inwoner/pull/1014](https://github.com/maykinmedia/open-inwoner/pull/1014) because the two are closely related.

Didn't know about VCR. Looks nice and I'd be in favour of adopting it, especially seeing how I missed `mocks.simple`.

Reply:
Double-checked: I branched off `upgrade/django-admin-index` instead of develop when starting on this. The PR has been merged though.